src/DGtal/dec/doc/moduleHeatLaplaceOperator.dox

```diff
@@ -0,0 +1,175 @@
+\f$p \in C^\infty(\mathbb{R}^+ \times \partial M \times \partial M)\f$ is the
+heat kernel \cite rosenberg.
+
+There is a wide range of studies on the behavior of \f$p\f$ when \f$t\f$ tends
+to zero (called small time asymptotics of diffusion process). Early work
+includes the famous Varadhan formula \cite varadhan on closed manifolds with or
+without borders later extended by Molchanov \cite molchanov on a wider class of
+shapes: \f[ p(t,x,y) \mathrel{\mathop{\sim}\limits_{t \rightarrow 0}} \frac{e^{-
+\frac{d(x,y)^2}{4 t}}}{(4\pi t)^{\frac{d}{2}}} \f] where \f$d(\cdot,\cdot)\f$
+corresponds to the intrinsic geodesic distance. This approximation is not robust
+in practice and very sensitive to both geodesic distance approximation and
+numerical errors \cite Crane2013. Fortunately we know from Belkin et al. \cite BelkinN08 
+that in small time asymptotics, the geodesic
+distance can be approximated by the \f$l_2\f$ distance: \f[ p(t,x,y)
+\mathrel{\mathop{\sim}\limits_{t \rightarrow 0}} \tilde{p}(t,x,y) := \frac{e^{ -
+\frac{||x - y||^2}{4t} }}{(4 \pi t)^{\frac{d}{2}}}, \f] which leads to the
+following approximated solution of the heat equation: \f[ g(x,t) = \int_{y \in
+\partial M} \tilde{p}(t,x,y) u(y)dy. \f]
+
+By injecting the approximation into the heat equation we obtain: \f[ \Delta g(x,
+t) = \frac{\partial}{\partial t} \int_{ y\in \partial M } \tilde{p}(t, x, y)
+u(y)dy. \f] Using a finite difference on \f$t\f$, and the basic property that
+the integral of the heat kernel must be one: \f[ \Delta g(x, t) = \lim\limits_{t
+\rightarrow 0} \frac{1}{t} \int_{y\in \partial M } \tilde{p}(t, x, y) (u(y) -
+u(x))dy. \f]
+
+\section secDiscre The Discretization on Digital Surfaces
+
+We adapt the formulation of Belkin et al. on digital surfaces (see \cite CaissardDGCI2017).
+In the continuous heat kernel formulation, the parameter \f$t\f$ must tend to zero. 
+On digital surfaces, we set \f$t\f$ as a function of the grid step \f$h\f$, denoted
+\f$t_h\f$, that tends to zero as \f$h\f$ tends to zero.
+
+@note Typically the time parameter is set to \f$t_h = k \cdot h^{\alpha}\f$ with
+\f$\alpha\f$ positive and \f$k\f$ is a constant depending on the input shape.
+
+The discretization on digital surfaces is \f[ (\Delta_{DIGI} \, f)({\mathbf{s}})
+:= \frac{1}{t_h(4 \pi t_h)^{\frac{d}{2}}} \sum_{\mathbf{r}\in \mathbb{F}^d }
+e^{- \frac{||\dot{\mathbf{r}}
+        - \dot{\mathbf{s}}||^2}{4t_h}}  [f(\dot{{\mathbf{r}}})
+    - f(\dot{{\mathbf{s}}})]\mu(\mathbf{r}), \f] where \f$\mathbb{F}^d\f$ is the
+      set of elements dimension \f$d\f$ (for example surfels in a 2D surface
+      embedded in 3D), \f$\dot{\mathbf{r}}\f$ (resp. \f$\dot{\mathbf{s}}\f$) is
+      the centroid of the surfel \f$\mathbf{r}\f$ (resp. \f$\mathbf{s}\f$), and
+      \f$\mu(\mathbf{r})\f$ is equal to the dot product between an estimated
+      normal and the trivial normal orthogonal to the surfel \f$\mathbf{s}\f$.
+
```

Comment:
you should add a note saying that $alpha=1/3$ as in II would do the job and cite the JMIV paper (arxiv?HAL?) with the formal details and proofs

Reply:
I uploaded the paper to HAL as a preprint and waiting for their approval.